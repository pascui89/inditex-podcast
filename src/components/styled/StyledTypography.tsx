import {
    Typography,
    styled,
  } from '@mui/material';

export const StyledTypography = styled(Typography)`
    display: flex;
    font-family: monospace;
    font-weight: 700;
    letter-spacing: 0.3rem;
    color: inherit;
    text-decoration: none;
    cursor: pointer;
    margin-left: 16px;

    &:hover {
    color: #ff0000; /* Cambia el color de resaltado al pasar el ratón */
    }

    &::selection {
    background-color: #ff0000; /* Cambia el color de fondo de la selección */
    color: #ffffff; /* Cambia el color del texto de la selección */
}`;