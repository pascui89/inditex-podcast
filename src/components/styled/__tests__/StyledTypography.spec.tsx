import React from 'react';
import { render } from '@testing-library/react';
import { StyledTypography } from '../StyledTypography';
import '@testing-library/jest-dom/extend-expect';

global.React = React;

describe('StyledTypography', () => {

  it('should render', () => {
    const { container } = render(<StyledTypography/>);
    expect(container).toMatchSnapshot();
  });
});
