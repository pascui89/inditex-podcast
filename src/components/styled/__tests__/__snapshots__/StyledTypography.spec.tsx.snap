// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledTypography should render 1`] = `
<div>
  <p
    class="MuiTypography-root MuiTypography-body1 css-m0d3b0-MuiTypography-root"
  />
</div>
`;
